4 reports.sql

```diff
@@ -0,0 +1,42 @@
+--The reports I need are as following:
+-- 1) How many people are booked per flight as: count of people, flight number, departure airport, when the flight is departing and destination.
+select p.FlightNumber, P.DepartureAirport, p.TimeDeparting, p.ArrivalAirport, AmountOfPassengers = count(*)
+from Passenger p 
+group by p.FlightNumber, P.DepartureAirport, p.TimeDeparting, p.
```

Comment:
Its seems that the arrival airport got mistakenly deleted in the group by

Reply:
fixed